cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -129,13 +131,6 @@ def pytest_generate_tests(metafunc):
             if depot_type == "ftp" and machine_id not in machines_ftp:
                 machines_ftp[machine_id] = credentials
                 machines_ftp[machine_id]["hostname"] = hostname
-
-            assert "use_for_log_collection" in depot_type_content,\
```

Comment:
Thought again and found a reason to leave it. 
I removed it because in cfme_data all depots are used for testing. But some of this depots are used in another test. So there is a possibility that some new items will be added in future.